documentation/jetty-documentation/src/main/asciidoc/programming-guide/server/server.adoc

```diff
@@ -36,3 +36,4 @@ include::http3/server-http3.adoc[]
 include::sessions/sessions.adoc[]
 include::websocket/server-websocket.adoc[]
 include::server-io-arch.adoc[]
+include::conscrypt/server-conscrypt.adoc[]
```

Comment:
Same as for the client, I think this should be moved to the `pg-server-http-connector-protocol` section, perhaps as a child, with all the subsections that configure TLS referring to this new child.